portfolio/src/main/webapp/index.html

```diff
@@ -1,31 +1,65 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Dhruvi's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>Dhruvi's Portfolio</h1>
-      <p>This is Dhruvi's portfolio.</p>
-      <p>Hi there! My name is Dhruvi and I am a STEP intern @ Google! I study CS at Rutgers University-New Brunswick!</p>
-      <div class="icons"> 
-            <a href="https://www.linkedin.com/in/dhruvi-sheth">
-                <img alt="Linkedin" src="images/linkedin.png" width="30" height="25">
-            </a>
-      </div>
-      <div class="icons">
-            <a href="https://github.com/dhruvi618">
-                <img alt="Github" src="images/github.png" width="25" height="25">
-            </a>
-      </div>
-      </br>
-      </br>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>Dhruvi Sheth</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <div class="navbar">
+            <a class="active" href="#home">Home</a>
+            <a href="#">About Me</a>
+            <a href="#">Experience</a>
+            <a href="#">Projects</a>
+            <a href="#">Contact</a>
+        </div>
+        <section id="home">
+            <div>
+                <h1>I'm Dhruvi Sheth.</h1>
+                </br>
+                <h3>Student | Aspiring Software Engineer | Tech Enthusiast</h3>
+                <a href="#">Learn More</a>
+            </div>
+        </section> 
+        <section id="exp">
+            <div class="experience">
+                <img src="images/google.png" alt="Google" width="100" height="100">
+                <h2>Google</h2>
+                <p>STEP Intern</p>
+                <p>Web App</p>
+            </div>
+            <div class="experience">
+                <img src="images/rutgers.png" alt="Rutgers" width="100" height="100">
+                <h2>Rutgers University</h2>
+                <p>Technology Intern</p>
+                <p>Handshake</p>
+            </div>
+            <div class="experience">
+                <img src="images/rutgers.png" alt="iCIMS" width="100" height="100">
+                <h2>iCIMS</h2>
+                <p>SWE Intern</p>
+                <p>Java Backend</p>      
+            </div>
+        </section>
+        <section id="contact">   
+            <div class="icons"> 
+                <a href="https://www.linkedin.com/in/dhruvi-sheth">
+                    <img alt="Linkedin" src="images/linkedin.png" width="30" height="25">
```

Comment:
😄  Nice job including an alt-text here. Can you expand it a little bit to explain more precisely where this link goes. Maybe "Dhruvi Sheth's LinkedIn Profile"

Reply:
Need to git push?

Reply:
just pushed the changes